node/src/accountant/mod.rs

```diff
@@ -578,6 +572,15 @@ impl Accountant {
         info!(self.logger, "Accountant bound");
     }
 
+    fn handle_config_change_msg(&mut self, msg: ConfigChangeMsg) {
+        if let ConfigChange::UpdateWallets(wallet_pair) = msg.change {
+            self.earning_wallet = wallet_pair.earning_wallet;
+            self.consuming_wallet_opt = Some(wallet_pair.consuming_wallet);
+        } else {
+            trace!(self.logger, "Unexpected message received: {:?}", msg);
```

Comment:
If this condition is of concern, it ought to be stronger than a `trace!`. If it's commonplace, and happens whenever the UI sends a particular kind of message, there probably shouldn't be any log at all here.